priam/src/main/java/com/netflix/priam/defaultimpl/PriamConfiguration.java

```diff
@@ -207,6 +208,7 @@
     private final String DEFAULT_PARTITIONER = "org.apache.cassandra.dht.RandomPartitioner";
     public static final String DEFAULT_AUTHENTICATOR = "org.apache.cassandra.auth.AllowAllAuthenticator";
     public static final String DEFAULT_AUTHORIZER = "org.apache.cassandra.auth.AllowAllAuthorizer";
+    public static final String DEFAULT_COMMITLOG_PROPS_FILE = "/conf/commitlog_archiving.properties";
```

Comment:
It's prefixed runtime with getCassHome() when anything gets the file's location like it was before in the StandardTuner class.
I got the idea from getYamlLocation() function, to do it like this, only modification was that I moved the string part into a static default String.
Of course I change the patch, if you have any advice, I just didn't want to set the value of a default string in the constructor, because getCassHome is not a static function.